module/x/gravity/keeper/querier_test.go

```diff
@@ -142,6 +146,9 @@ func TestAllValsetConfirmsBynonce(t *testing.T) {
 				return
 			}
 			require.NoError(t, err)
+			sort.Slice(got.Confirms, func(i, j int) bool {
```

Comment:
Why did you start sorting, because of the random addresses?

Reply:
Yes, because the serialized version would move around